src/main/java/PooPoo/PooPoo.java

```diff
@@ -0,0 +1,102 @@
+package PooPoo;
+
+import PooPoo.ui.*;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class PooPoo {
+    public static TaskList taskList = new TaskList();
+
+    public static void handleTask(String command) throws PooPooException {
```

Comment:
this function is quite long, splitting up into separate methods might help readability